build.sbt

```diff
@@ -40,7 +40,7 @@ lazy val article_api = (project in file(".")).
       "javax.servlet" % "javax.servlet-api" % "3.1.0" % "container;provided;test",
       "org.scalatra" %% "scalatra-json" % Scalatraversion,
       "org.scalatra" %% "scalatra-scalatest" % Scalatraversion % "test",
-      "org.json4s"   %% "json4s-native" % "3.3.0",
+      "org.json4s"   %% "json4s-native" % "3.5.0",
```

Comment:
Lette etter en changelog for 3.5 men fant ingen; er det noe spennende som har skjedd siden 3.3.0 du vet om? :)